funnel/forms/label.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+from baseframe import __
+import baseframe.forms as forms
+
+__all__ = ['LabelForm', 'LabelOptionForm']
+
+
+class LabelForm(forms.Form):
+    id = forms.IntegerField("", widget=forms.HiddenInput(), validators=[forms.validators.Optional()])
```

Comment:
Integer id should never be exposed to the front-end. If this is unavoidable, use UUID primary keys.

Reply:
Use the `huuid` column represented as a StringField to get a hex representation without dashes.

Reply:
using `name` field instead.